scripts/common_test.py

```diff
@@ -36,17 +36,28 @@
 from core import utils
 from core.tests import test_utils
 
+import github
 from typing import Generator, List, NoReturn
 from typing_extensions import Literal
 
 from . import common
 
-_PARENT_DIR = os.path.abspath(os.path.join(os.getcwd(), os.pardir))
-_PY_GITHUB_PATH = os.path.join(
-    _PARENT_DIR, 'oppia_tools', 'PyGithub-%s' % common.PYGITHUB_VERSION)
-sys.path.insert(0, _PY_GITHUB_PATH)
 
-import github # isort:skip  pylint: disable=wrong-import-position
+# The pool_size argument is required by Requester.__init__(), but it is
+# missing from the typing definition in Requester.pyi.  We therefore
```

Comment:
Can we just update this definition (either in our repo or upstream)?

Reply:
We can't update Requester.pyi directly (it's a third-party package). This has already been reported to the upstream maintainers by way of a PR, but it lapsed and was closed: PyGithub/PyGithub#2151. Unfortunately it doesn't look like PyGithub is very actively maintained given that they have a pinned issue titled "Is the PyGithub project dead? ..." (PyGithub/PyGithub#2178) with no response from the maintainers (except to pin the issue)

Reply:
Ah ok, thanks. Could you please add this context to the comment? (You can just say "here is a PR attempting to fix this in PyGithub, but it lapsed because the project doesn't seem very active and was closed.") or similar.

That way other folks reading this code have context on why we're leaving it as-is.

Reply:
done